src/pretty/func_call.rs

```diff
@@ -83,7 +83,7 @@ impl<'a> ParenthesizedFuncCallArg<'a> {
 impl<'a> PrettyPrinter<'a> {
     pub(super) fn convert_func_call(&'a self, func_call: FuncCall<'a>) -> ArenaDoc<'a> {
         let mut doc = self.convert_expr(func_call.callee());
-        if let Some(res) = self.check_disabled(func_call.args().to_untyped()) {
+        if let Some(res) = self.check_comment_unable(func_call.args().to_untyped()) {
```

Comment:
`check_comment_unable` is a super weird name.

Reply:
I struggled with naming here. I want to convey that the node is set as unformatteable and unable to format due to insufficient capability.

Reply:
How about `check_unformattable`?